content/en/docs/reference/using-api/deprecation-guide.md

```diff
@@ -0,0 +1,265 @@
+
+#### NetworkPolicy
+
+The **extensions/v1beta1** API version of NetworkPolicy is no longer served as of v1.16.
+
+* Migrate manifests and API clients to use the **networking.k8s.io/v1** API version, available since v1.8.
+* All existing persisted objects are accessible via the new API
+
+#### DaemonSet
+
+The **extensions/v1beta1** and **apps/v1beta2** API versions of DaemonSet are no longer served as of v1.16.
+
+* Migrate manifests and API clients to use the **apps/v1** API version, available since v1.9.
+* All existing persisted objects are accessible via the new API
+* Notable changes:
+    * `spec.templateGeneration` is removed
+    * `spec.selector` is now required and immutable after creation; use the existing template labels as the selector for seamless upgrades
+    * `spec.updateStrategy.type` now defaults to `RollingUpdate` (the default in `extensions/v1beta1` was `OnDelete`)
+
+#### Deployment
+
+The **extensions/v1beta1**, **apps/v1beta1**, and **apps/v1beta2** API versions of Deployment are no longer served as of v1.16.
+
+* Migrate manifests and API clients to use the **apps/v1** API version, available since v1.9.
+* All existing persisted objects are accessible via the new API
+* Notable changes:
+    * `spec.rollbackTo` is removed
+    * `spec.selector` is now required and immutable after creation; use the existing template labels as the selector for seamless upgrades
+    * `spec.progressDeadlineSeconds` now defaults to `600` seconds (the default in `extensions/v1beta1` was no deadline)
+    * `spec.revisionHistoryLimit` now defaults to `10` (the default in `apps/v1beta1` was `2`, the default in `extensions/v1beta1` was to retain all)
+    * `maxSurge` and `maxUnavailable` now default to `25%` (the default in `extensions/v1beta1` was `1`)
+
+#### StatefulSet
+
+The **apps/v1beta1** and **apps/v1beta2** API versions of StatefulSet are no longer served as of v1.16.
+
+* Migrate manifests and API clients to use the **apps/v1** API version, available since v1.9.
+* All existing persisted objects are accessible via the new API
+* Notable changes:
+    * `spec.selector` is now required and immutable after creation; use the existing template labels as the selector for seamless upgrades
+    * `spec.updateStrategy.type` now defaults to `RollingUpdate` (the default in `apps/v1beta1` was `OnDelete`)
+
+#### ReplicaSet
+
+The **extensions/v1beta1**, **apps/v1beta1**, and **apps/v1beta2** API versions of ReplicaSet are no longer served as of v1.16.
+
+* Migrate manifests and API clients to use the **apps/v1** API version, available since v1.9.
+* All existing persisted objects are accessible via the new API
+* Notable changes:
+    * `spec.selector` is now required and immutable after creation; use the existing template labels as the selector for seamless upgrades
+
+## What To Do
+
+Kubernetes 1.22 will be released later in 2021, so be sure to audit
+your configuration and integrations now!
```

Comment:
This is more appropriate for a blog article than for a reference. (This wording is 100% guaranteed to become stale).

If we want to call out the next release we do have shortcodes that let you specify what the next minor release will be and have Hugo update that. There's no shortcode for the year that release happens in though.

Reply:
dropped the immediately stale text, left the specific tasks